### PR TITLE
chore!: reinstate 2 min blocks for dibbler

### PR DESCRIPTION
**Pull request by sdbondi**
Description
---

- Updates dibble consensus to 2-minute blocks
- Updates dibbler genesis timestamp

Motivation and Context
---
Dibbler block times are too fast

How Has This Been Tested?
---

### CODE REVIEW

File: base_layer/core/src/consensus/consensus_constants.rs [hunk 1]
```diff
@@ -322,13 +322,13 @@ impl ConsensusConstants {
             max_target_time: 1800,
             min_difficulty: 60_000.into(),
             max_difficulty: u64::MAX.into(),
-            target_time: 30,
+            target_time: 300,
         });
         algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
             max_target_time: 1200,
             min_difficulty: 60.into(),
```
Comment (stringhandler) [inline base_layer/core/src/consensus/consensus_constants.rs]: Need this updated as well

Comment (sdbondi) [reply base_layer/core/src/consensus/consensus_constants.rs]: Nice catch

File: base_layer/core/src/consensus/consensus_constants.rs [hunk 2]
```diff
@@ -322,13 +322,13 @@ impl ConsensusConstants {
             max_target_time: 1800,
             min_difficulty: 60_000.into(),
```
Comment (delta1) [inline base_layer/core/src/consensus/consensus_constants.rs]: I think this is supposed to be 60m?

```suggestion
            min_difficulty: 60_000_000.into(),
```

Comment (delta1) [reply base_layer/core/src/consensus/consensus_constants.rs]: or was it 6m in weatherwax?

Comment (sdbondi) [reply base_layer/core/src/consensus/consensus_constants.rs]: Pretty sure it's 60_000 in weatherwax 
I think somewhere along the lines weatherwax's values have changed (30s block time definitely isn't correct)

image

**Comment by delta1** [comment]
Previous weatherwax constants 

image

File: base_layer/core/src/consensus/consensus_constants.rs (hunk 2 above)
Comment (delta1) [reply base_layer/core/src/consensus/consensus_constants.rs]: I think those got altered during the validator node merge

**Comment by sdbondi** [comment]
@delta1 Ah you're right - thanks

File: base_layer/core/src/consensus/consensus_constants.rs (hunk 2 above)
Comment (sdbondi) [reply base_layer/core/src/consensus/consensus_constants.rs]: Yeah

File: base_layer/core/src/consensus/consensus_constants.rs [hunk 3]
```diff
@@ -249,9 +249,9 @@ impl ConsensusConstants {
         // setting sha3/monero to 40/60 split
         algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
             max_target_time: 180,
```
Comment (delta1) [inline base_layer/core/src/consensus/consensus_constants.rs]:
```suggestion
            max_target_time: 1800,
```

**Review by delta1 — COMMENTED** [review]
Looks good just the max_target_time from weatherwax

File: base_layer/core/src/consensus/consensus_constants.rs (hunk 3 above)
Comment (sdbondi) [reply base_layer/core/src/consensus/consensus_constants.rs]: lol `// TODO: reset to previous value` - thanks

**Review by stringhandler — APPROVED** [review]

**Pull request merged**